index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CSS Backgrounds and Gradients</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="css/styles.css" />
+  </head>
+  <body>
+    <main>
+      <div class="header">
+        <nav id="navbar">
+          <ul>
+            <li>
+              <a href="#"><i class="fa fa-apple" aria-hidden="true"></i></a>
+            </li>
+            <li><a href="#">Store</a></li>
+            <li><a href="#">Mark</a></li>
+            <li><a href="#">ipod</a></li>
+            <li><a href="#">iphone</a></li>
+            <li><a href="#">Ipad</a></li>
+            <li><a href="#">Itunes</a></li>
+            <li><a href="#">Support</a></li>
+            <li>
+              <i class="fa fa-search"></i> <input class="search" type="text" />
+            </li>
+          </ul>
+        </nav>
+        <div class="info">
+          <p class="big-text">What will your verse be?</p>
+          <p class="p1">
+            Everyone has something to share. See how people are using <br />
+            iPad in their unique ways to add to the world's story
+          </p>
+          <p class="p1">
+            Watch the "Your Verse" film and explore the stories
+            <i class="fa fa-play"></i>
+          </p>
+        </div>
+      </div>
+      <div class="phones">
+        <div><img src="img/image4.jpg" alt="image" /></div>
+        <div><img src="img/image3.jpg" alt="image" /></div>
+        <div><img src="img/image2.jpg" alt="image" /></div>
+        <div><img src="img/image1.jpg" alt="image" /></div>
```

Comment:
Kindly set each div with a class and give them the background images and add a link to each div. The focus for this project is using background and gradients

Reply:
Let us implement the DRY rule so we avoid repeating code in our CSS file, we should also avoid using ids and tag names for styling